npm/qsharp/src/compiler/compiler.ts

```diff
@@ -240,6 +240,7 @@ export class Compiler implements ICompiler {
     eventHandler: IQscEventTarget,
```

Comment:
Actually, maybe not if this API didn't change. We should check they are only using the ICompiler interface, and not calling any wasm exports directly (which they shouldn't be)

Reply:
Yeah, that's what I was assuming. You are right that it could be a breaking change for anything calling directly into WASM though.

Reply:
I double checked the calls there, and nothing goes directly to WASM, it all uses the compiler from the npm package. So we should be good!